index.html

```diff
@@ -895,13 +895,13 @@ <h3 id="event-touchcancel">The <dfn class="event"><code>touchcancel</code></dfn>
       </p>
 
       <p class="note">
-        User agents will typically dispatch mouse and click events when there is only a single
-        <a>active touch point</a>. Multi-touch interactions – involving two or more
+        User agents will typically dispatch mouse and click events only for single finger activation gestures (like tap and long press), and only when there is a single
```

Comment:
nit: the "and only when..." is now redundant with "single finger", right?  Could just remove the 2nd part entirely?
Also should "single finger" be hyphenated?